### PR TITLE
0.3.0

### PR DESCRIPTION
**Pull request by mediumTaj**

### CODE REVIEW

**Comment by mediumTaj** [comment]
@germanattanasio look at the 0.3.0 release

File: Scenes/ServiceExamples/Scripts/ExampleSpeechToText.cs [hunk 1]
```diff
@@ -0,0 +1,29 @@
...
+		if (result != null && result.Results.Length > 0)
+		{
+			foreach( var res in result.Results )
+			{
+				foreach( var alt in res.Alternatives )
+				{
+					string text = alt.Transcript;
+					Debug.Log(string.Format( "{0} ({1}, {2:0.00})\n", text, res.Final ? "Final" : "Interim", alt.Confidence));
+				}
+			}
```
Comment (mediumTaj) [inline Scenes/ServiceExamples/Scripts/ExampleSpeechToText.cs]: take a look at this

**Comment by germanattanasio** [comment]
@mediumTaj we need @rolyle  and @ereneld to sign the CLA

**Comment by mediumTaj** [comment]
@germanattanasio I think we are good now... everybody has signed the CLA. Let me know if anything looks off.

**Comment by mediumTaj** [comment]
Oops, forgot the changelog... added.

**Comment by germanattanasio** [comment]
I didn't see the CHANGELOG.md did you push it to `develop` ?

**Comment by germanattanasio** [comment]
@mediumTaj: I've removed a few more `.meta` files.
If everything is OK we can merge this with master after doing a release.

**Comment by mediumTaj** [comment]
OK sounds good

**Pull request merged**